.github/workflows/ci.yml

```diff
@@ -35,9 +35,19 @@ jobs:
             run_regression_args: --tester base --tester model --tester synth --tester abduct --tester dump
  
           - name: macos:production-arm64
+            os: macos-14
+            config: production --auto-download --all-bindings --editline
+            cache-key: production-arm64
+            strip-bin: strip
+            python-bindings: true
+            check-examples: true
+            exclude_regress: 3-4
+            run_regression_args: --tester base --tester model --tester synth --tester abduct --tester dump
+
+          - name: macos:production-arm64-mingw
```

Comment:
Maybe let's call this `-cross` instead of `-mingw`, since the latter is Windows-specific.